multicluster/service-mirror/cluster_watcher.go

```diff
@@ -894,6 +894,10 @@ func (rcsw *RemoteClusterServiceWatcher) Start(ctx context.Context) error {
 					rcsw.log.Debugf("skipped processing endpoints object %s/%s: missing %s label", epNew.Namespace, epNew.Name, consts.DefaultExportedServiceSelector)
 					return
 				}
+				if rcsw.isRemoteDiscovery(epNew.Labels) {
+					rcsw.log.Debugf("skipped processing endpoints object %s/%s (not mirrored in remote-discovery mode)", epNew.Namespace, epNew.Name)
```

Comment:
might reword this message because it almost sounds like it's saying that this service is not in remote-discovery mode.